### PR TITLE
Share Site: Adjust copy based on feedback

### PR DESCRIPTION
**Pull request by danielbachhuber**
Fixes https://github.com/Automattic/dotcom-forge/issues/2593

## Proposed Changes

Adjusts the copy used in the Share Site for Preview components, and adds an inline support link:

### Sites Page Modal

image

image

### Privacy Settings, Launched

image

image

### Privacy Settings, Unlaunched

image

image

## Testing Instructions

1. Create a new WordPress.com Business site.
2. Navigate to Sites, click on the ellipsis menu for your new site, click on 'Share site for preview'
3. Verify the modal appears and works as expected.
4. Navigate to General Settings for the site.
5. Verify the share site component appears as expected.

### CODE REVIEW

**Comment by kozer** [comment]
Great work @danielbachhuber ! It works as expected.

One small note:
When the user is on Sites and the modal is open, when he clicks on "Learn more" a new modal opens. If the user closes the last modal, I would expect to show the previous one instead of having to go back to the ellipsis menu and reopen the modal.

It's not related to that ticket, so it's ok to skip that. If this is the case, it would be great if we create a new issue for that.

**Review by kozer — APPROVED** [review]

**Review by wojtekn — APPROVED** [review]
It's a helpful change! The code looks good and works as expected.

As we changed how the popup content is structured, it would make sense to change the loading state of the component:
- move the new text out of the loading indicator scope to show before we start loading data
- change Spinner to LoadingBar
We could do it as follow-up  PR, though.

In addition to @kozer note, I see one more annoying thing related to the 'Learn more' link - when I access it on the General Settings page, a window scrolls to the top.

**Comment by javierarce** [comment]
Looking at the text again, I think we could shorten it a bit more: 


> 'Coming Soon' sites are only visible to you and invited users. Enable 'Share site' to let collaborators without an account view your site.

File: client/components/site-preview-link/index.tsx [hunk 1]
```diff
@@ -106,17 +107,24 @@ export default function SitePreviewLink( {
 	}
 	return (
 		<div>
+			<p>
+				{ translate(
+					"Sites in 'Coming Soon' mode are visible only to you and to the logged-in users you invite. Toggle 'Share site' to allow collaborators without an account to see your site. {{inlineSupportLink}}Learn more.{{/inlineSupportLink}}",
```
Comment (simison) [inline client/components/site-preview-link/index.tsx]: Should `'`s be `"`s instead?

Comment (simison) [reply client/components/site-preview-link/index.tsx]: "Coming Soon" in Title Case was also odd.

Comment (javierarce) [reply client/components/site-preview-link/index.tsx]: Good point, yes, we should use double quotes.

Comment (danielbachhuber) [reply client/components/site-preview-link/index.tsx]: Updated with cb2165420297108adb1337107aad092f16230901

**Comment by danielbachhuber** [comment]
> As we changed how the popup content is structured, it would make sense to change the loading state of the component:
> 
>     * move the new text out of the loading indicator scope to show before we start loading data
> 
>     * change Spinner to LoadingBar
>       We could do it as follow-up  PR, though.

Changed this behavior with 010fba495c1bfbe238bd7259e4b8ddc497f24676



> When the user is on Sites and the modal is open, when he clicks on "Learn more" a new modal opens. If the user closes the last modal, I would expect to show the previous one instead of having to go back to the ellipsis menu and reopen the modal.
>
> [...]
>
> In addition to @kozer note, I see one more annoying thing related to the 'Learn more' link - when I access it on the General Settings page, a window scrolls to the top.

Yeah, both of those are kind of annoying. I haven't figured out where they're coming from/how to fix yet.

@javierarce Do you think we should leave the "Learn more" in place or remove it?

**Comment by javierarce** [comment]
> @javierarce Do you think we should leave the "Learn more" in place or remove it?

I think we could remove those links, the description and the text underneath the input field should be enough.

**Comment by danielbachhuber** [comment]
> I think we could remove those links, the description and the text underneath the input field should be enough.

@javierarce Removed with 08036d80c9becca836e36d7e0aac62459f47781b

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8019144

Thank you @danielbachhuber for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8019144).

**Pull request merged**